src/actor/disk.rs

```diff
@@ -186,6 +198,6 @@ impl super::Antagonist for DiskActor {
 
         sleep_random_ms(100).await;
 
-        result
+        result.map_err(|e| e.into())
```

Comment:
IIRC Clippy will complain about this when we next run it--it'll want either a `?` operator or a direct reference to `into`:

```suggestion
        result.map_err(Into::into)
```

Reply:
done in dd31884